estate/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "estate",
+    "license": "AGPL-3",
+    "depends": ["base"],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/estate_property_actions.xml",
+        "views/estate_property_type_actions.xml",
+        "views/estate_property_tag_actions.xml",
+        "views/estate_property_menus.xml",
+    ],
+    "application": True,
+    "installable": True,
+    "auto_install": False,
```

Comment:
This is the default value (obviously, you don't want all modules to be auto-installed) so you don't have to define it